baseapp/abci.go

```diff
@@ -491,6 +499,11 @@ func (app *BaseApp) OfferSnapshot(req abci.RequestOfferSnapshot) abci.ResponseOf
 
 // ApplySnapshotChunk implements the ABCI interface. It delegates to app.snapshotManager if set.
 func (app *BaseApp) ApplySnapshotChunk(req abci.RequestApplySnapshotChunk) abci.ResponseApplySnapshotChunk {
+	if app.snapshotManager == nil {
+		app.logger.Error("No configured snapshot manager")
```

Comment:
how about _"Snapshot Mangager not configured"_

Reply:
just no punctuation please 🙏